resources/views/page/amp.blade.php

```diff
@@ -1,6 +1,11 @@
 @extends('support::amp')
 
 @section('content')
+    {{-- DO NOT REMOVE - page identity tag --}}
+    <!-- P:{{ $page->id }} C:{{ $child_page->id ?? '0' }} GC:{{ $grand_child_page->id ?? '0' }} -->
```

Comment:
Example of an identity tag that should always remain to allow `assertSee(...)` in tests.  The "DO NOT REMOVE" line is a blade comment, so that won't render in the html.